### ISSUE TITLE
Rotate Picture

### ISSUE THREAD

**Issue by martinallnet** [ikorb/gcvideo#114]
Is it possible to add a option for picture rotate?

**Comment by ikorb** [comment]
No, to rotate the image a full frame must be buffered, but the FPGA does not have enough memory to do so.
